samples/notebooks/dotnet/08-chatGPT-with-DALL-E-2.ipynb

```diff
@@ -99,10 +99,20 @@
     "var (useAzureOpenAI, model, azureEndpoint, apiKey, orgId) = Settings.LoadFromFile();\n",
     "\n",
     "// Configure the two AI features: OpenAI Chat and DALL-E 2 for image generation\n",
-    "var kernel = new KernelBuilder()\n",
-    "    .WithOpenAIChatCompletionService(\"gpt-3.5-turbo\", apiKey, orgId)\n",
-    "    .WithOpenAIImageGenerationService(apiKey, orgId)\n",
-    "    .Build();\n",
+    "var builder = new KernelBuilder();\n",
+    "\n",
+    "if(useAzureOpenAI)\n",
+    "{\n",
+    "    builder.WithAzureChatCompletionService(\"gpt-3.5-turbo\", azureEndpoint, apiKey);\n",
```

Comment:
In Azure OpenAI (as opposed to plain OpenAI), "gpt-3.5-turbo" is named without a period:
"gpt-35-turbo"

Reply:
thanks. just fixed it